[DOCU-2517] App Reg Multiple RGs and DCR - GA release (#4748)
Kong/docs.konghq.com: app/konnect/updates.md

## November 2022

### 2022.11.14

**Multiple runtime groups**

> : All the runtime groups in Konnect including default and non default runtime groups will be eligible for application registration.

Reviewer: ```suggestion
: All the runtime groups in {{site.konnect_short_name}}, including default and non-default runtime groups, are eligible for application registration.
```
